contracts/modules/v2/Claims.sol

```diff
@@ -86,17 +87,18 @@ contract Claims is IClaims, MasterAwareV2 {
     uint nxmPriceInPayoutAsset = poolContract.getTokenPrice(payoutAsset);
     uint nxmPriceInETH = poolContract.getTokenPrice(0);
 
-    // Calculate the expected in NXM using the NXM price at cover purchase time
+    // Calculate the expected payout in NXM using the NXM price at cover purchase time
     uint expectedPayoutInNXM = requestedAmount * PRECISION / nxmPriceInPayoutAsset;
 
     // Determine the total rewards that should be minted for the assessors based on cover period
-    uint totalReward = max(
-      config.maxRewardNXM * PRECISION,
-      expectedPayoutInNXM * config.rewardRatio * coverPeriod / 365 days / RATIO_BPS
+    uint totalReward = min(
+      uint(config.maxRewardNXM) * PRECISION,
```

Comment:
would be more clear if maxRewardNXM is actually stored as 50e18 because you generally assume all token amounts are expressed in wei on-chain, and when we have things like the  cover amount expressed in units instead of wei it's prone to mistakes

Reply:
Would have made sense for something such as cover. This is only used internally and not meant as some piece of data other contracts rely on. The only time where user input could potentially be wrongly introduced is when the parameter is updated through governance. However given that it's an uint16 it would just revert for values > 65535, so passing 1e18 by mistake would not be possible, not even in the UI. Some methods require multiple parts of config and reading everything in one SLOAD saves gas.

Reply:
it's more about interpreting it as wei, when it's actually units

The default for any token amounts stored on-chain is to store the amount of wei, at the very let's look for a better name to indicate it's units not wei